.github/workflows/update-deps.yaml

```diff
@@ -22,13 +22,19 @@ env:
 jobs:
   update-deps:
     runs-on: ubuntu-latest
+    container:
+      image: gcr.io/istio-testing/build-tools:master-8eb42e9551b9a67c330aeef783f2498647d91289
+      options: --entrypoint ''
 
     steps:
     - uses: actions/checkout@v4
       with:
         repository: istio/test-infra
         ref: master
 
+    # this is a workaround for a permissions issue when using the istio build container
+    - run: git config --system --add safe.directory /__w/sail-operator/sail-operator
```

Comment:
is this directory correct?

Reply:
yeah, not sure why it's sail-operator/sail-operator, but it's correct - and without it, the action fails